Test.C

```diff
@@ -11,108 +11,214 @@
+        {
+            printf("\nInsufficient funds");
+            return; //lazy return lol
+        }
+        printf("\nAre you sure? Y/N");
+        scanf(" %c", &yesNo);
+        if (yesNo == 'N' || yesNo == 'n')
+            return;
+        printf("\nThank You!");
+        player.currentCredits = player.currentCredits - transactionTotal;
+        player.equipmentInventory[equipmentselect - 1] = player.equipmentInventory[equipmentselect - 1] + amountSelect;
+    }
+    else
+    {
+        printf("Selection out of range");
+        return;
+    }
+}
+
+void encryptionStore()
+{
+    int encryptSelect;
+    int amountSelect;
+    int transactionTotal;
+    char yesNo;
+
+    printf("|-------------------------------- Encryption Store --------------------------------|\n");
+    printf("%s %s %d %f\n", encryptionStock[0][0] encryptionStock[0][1] encryptionStock[0][2] encryptionStock[0][3]);
+    printf("%s %s %d %f\n", encryptionStock[1][0] encryptionStock[1][1] encryptionStock[1][2] encryptionStock[1][3]);
+    printf("%s %s %d %f\n", encryptionStock[2][0] encryptionStock[2][1] encryptionStock[2][2] encryptionStock[2][3]);
+    printf("%s %s %d %f\n", encryptionStock[3][0] encryptionStock[3][1] encryptionStock[3][2] encryptionStock[3][3]);
+    printf("%s %s %d %f\n", encryptionStock[4][0] encryptionStock[4][1] encryptionStock[4][2] encryptionStock[4][3]);
+    printf("%s %s %d %f\n", encryptionStock[5][0] encryptionStock[5][1] encryptionStock[5][2] encryptionStock[5][3]);
+    printf("%s %s %d %f\n", encryptionStock[6][0] encryptionStock[6][1] encryptionStock[6][2] encryptionStock[6][3]);
+    printf("%s %s %d %f\n", encryptionStock[7][0] encryptionStock[7][1] encryptionStock[7][2] encryptionStock[7][3]);
+    printf("%s %s %d %f\n", encryptionStock[8][0] encryptionStock[8][1] encryptionStock[8][2] encryptionStock[8][3]);
+    printf("%s %s %d %f\n", encryptionStock[9][0] encryptionStock[9][1] encryptionStock[9][2] encryptionStock[9][3]);
+    printf("Please Select the number of the encryption you would like to buy");
+   
+    If(encryptSelect > 0 && encryptSelect < 11)
+    {
+        printf("You have selected: ");
+        printf("%s", printf("%s", encryptionStock[encryptSelect -1][0]);
+        printf("\n How many would you like to buy?");
+        scanf("\n%d", &amountSelect);
+        int transactionTotal = encryptionStock[encryptSelect -1][2] * amountSelect;
+        printf("\nThis will cost: %d ", transactionTotal);
+        if (player.currentCredits != transactionTotal)
+        {
+            printf("\nInsufficient funds");
+            return;
+        }
+        printf("\nAre you sure? Y/N");
+        scanf(" %c", &yesNo);
+        if (yesNo == 'N' || yesNo == 'n')
+            return;
+        printf("\nThank You!");
+        player.currentCredits = player.currentCredits - transactionTotal;
+        /*
+        So is the starting encryption head? Where does that get initalized?
```

Comment:
`player.encryptionInventory` is the root or head to the linked list. It is initialized with one element in the `// Setup player` section in `main`.